README.md

```diff
@@ -14,7 +14,7 @@ In this section, we show how to quickly run NGINX Prometheus Exporter for NGINX
 
 ### A Note about NGINX Ingress Controller
 
-If you’d like to use the NGINX Prometheus Exporter with [NGINX Ingress Controller](https://github.com/nginxinc/kubernetes-ingress/) for Kubernetes, see [this doc](https://github.com/nginxinc/kubernetes-ingress/blob/master/docs/installation.md#5-access-the-live-activity-monitoring-dashboard) for the installation instructions.
+If you’d like to use the NGINX Prometheus Exporter with [NGINX Ingress Controller](https://github.com/nginxinc/kubernetes-ingress/) for Kubernetes, see [this doc](https://docs.nginx.com/nginx-ingress-controller/logging-and-monitoring/prometheus/) for the installation instructions.
```

Comment:
I noticed that the link here was broken, I think this is the equivalent in the new docs